src/plugins/walletConnectToDapps/components/modals/EIP155TransactionConfirmation.tsx

```diff
@@ -94,6 +96,17 @@ export const EIP155TransactionConfirmation: FC<
     [fees, feeAsset, feeAssetPrice],
   )
 
+  const value = useMemo(() => {
+    if (!feeAsset) return '0'
+
+    const valueCryptoBaseUnit =
+      transaction?.value && isHex(transaction.value)
+        ? fromHex(transaction.value, 'bigint').toString()
+        : transaction?.value
+    const valueCryptoPrecision = fromBaseUnit(valueCryptoBaseUnit ?? '0', feeAsset.precision)
```

Comment:
`value` should always be denominated in wei and that was probably wrong the whole time